### ISSUE TITLE
Change pypi package requirement to tensorflow>=

### ISSUE THREAD

**Issue by kloppjp** [tensorflow/lattice#9]
Hello,

when I install tensorflow_lattice (_gpu), it triggers a reinstall of TensorFlow, although a newer version of TensorFlow is installed. I haven't performed extensive testing, however I can usually just uninstall that version of TF and reinstall a newer one. Is this hard version requirement necessary for compatibility? 
Relaxing it would make life a little easier ;)

Thank you for your great work!! :)

**Comment by si-you** [comment]
Hi! You're right about the compatibility. It was ">=" instead of "==", and when TF r1.4 was released, our pip package that worked for TF r1.3 stopped working. To be more precise, our C++ kernel stopped working with TF r1.4's C++ codes. We fixed it by re-compiling everything with TF r1.4 branch, and then added the "==" requirement to make sure the package is always working. The current package may not work with TF r1.5, and once TF r1.5 is released, we'll upgrade our pip package that can work with TF r1.5.

Does this answer your question?

Thank you!!
